docs/README.md

```diff
@@ -1,14 +1,84 @@
-# VTEX IO Store Component Template
+# Modal Layout
 
-This demo component shows you how to make a simple yet complete VTEX IO store component.
+WIP
 
-It shows a simple demonstration of the following features:
+:construction: :construction: :construction:
 
-- Declaring a component interface in `store/interfaces.json`.
-- Declaring a schema to receive props via Storefront.
-- Using the `messages` feature to have localized strings.
-- Using an NPM package like `react-intl`
-- Using another VTEX IO App like `vtex.styleguide`.
-- Using CSS Modules to make your component customizable via CSS on a store theme.
-- Using tachyons tokens to quickly style your components.
-- Creating tests for your VTEX IO app.
+*Do not use this repository until it has a stable version*
+
+
+## Blocks API
+
+The API for `modal-layout` and `modal-trigger` are very similar to `flex-layout`.
+
+```jsonc
+{
+  "modal-layout": {
+    "component": "Modal",
+    "composition": "children",
+    "allowed": "*"
+ },
+  "modal-trigger": {
+    "component": "ModalTrigger",
+    "composition": "children",
+    "allowed": "*"
+  }
+}
+
+```
+
+Notice that you could use _any_ array of blocks as `children`, given that they are allowed by the `block` that is directly above your `modal-layout`.
+
+## Configuration
+
+This props should be edited at your theme's `blocks.json`:
+
+### Modal
+
+| Prop name | Type | Description | Default value |
+| --- | --- | --- | --- |
+| `title` | `string`| The title of the modal | `undefined` |
+| `showCloseButton`| `boolean`| If it should show the close button | `true` |
+| `backdropInvisible` | `boolean` | If the Backdrop should have a transparent background (but if you click in the backdrop it still gonna close the modal) | `false` |
+| `hideBackdrop`| `boolean` | If you pass `true` it will not render the backdrop, which means that you won't see it and you can't click it to close the modal | `false`
```

Comment:
Change prop to `backdrop` and the values as an enum with `clickable`, `display`, `none`.